package.json

```diff
@@ -74,24 +74,28 @@
     "dev:watch": "watch 'npm run coverage' lib/ test/",
     "predev": "npm run setup",
     "dev": "npm-run-all -p --silent dev:*",
-    "prettier": "prettier --write --print-width=120 *.{js,ts,json,md} lib/*.js test/*.js bench/bench*",
```

Comment:
During the rebase I noticed that there's a print-width setting here that isn't actually followed for all files; the diff was actually smaller when going with the default 80, so I did that.